src/pages/integrations/bitpay-card/bitpay-card-intro/bitpay-card-intro.ts

```diff
@@ -111,14 +112,17 @@ export class BitPayCardIntroPage {
   }
 
   public orderBitPayCard() {
+    this.bitPayCardProvider.logEvent('legacycard_order', {});
     let url = 'https://bitpay.com/visa/get-started';
     this.externalLinkProvider.open(url);
   }
 
   public connectBitPayCard() {
     if (this.accounts.length == 0) {
+      this.bitPayCardProvider.logEvent('legacycard_connect', {});
       this.startPairBitPayAccount();
     } else {
+      this.bitPayCardProvider.logEvent('legacycard_connect', {});
```

Comment:
If the logEvent is the same as the `if`, maybe we could unify it and put it out of the `if else`